README.asciidoc

```diff
@@ -15,6 +15,14 @@ Install project deps:
 
  npm install
 
+Add git hooks:
+
+ mkdir -p .git/hooks
+ cat > .git/hooks/pre-push <<EOF
+ #!/bin/bash -e
+ git-hooks/pre-push
+ EOF
+
```

Comment:
This is the only part of this that I don't love—that each developer has to remember to run this. Perhaps it is more trouble than it is worth, but we could use something like this:
https://github.com/kilianc/shared-git-hooks
to enforce that this step isn't forgotten.